File: Online_Judge_Problems/BOJs/boj_11021/b11021.c
// AxB -7
// https://www.acmicpc.net/problem/11021

#include <stdio.h>
#include <stdlib.h>

int main() {
  int a, b;
  int t;

  scanf("%d", &t);

  int *results = (int *)malloc(t * sizeof(int));

  for (int i = 0; i < t; i++) {
    scanf("%d %d", &a, &b);
    results[i] = a + b;
  }

  for (int i = 0; i < t; i++) {
    printf("Case #%d: %d\n", i+1, results[i]);
  }

  free(results);

  return 0;
}

File: boj/boj_11021/b11021.js
// AxB -7
// https://www.acmicpc.net/problem/11021

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

let input = []

rl.on('line', (line) => {
  input.push(line)
})

rl.on('close', () => {
  const t = parseInt(input[0], 10);
  let results = []

  for (let i=1; i<=t; i++) {
    const [a, b] = input[i].split(' ').map(Number)
    results.push(a + b)
  }

  for (let i in results) {
    console.log(`Case #${parseInt(i)+1}: ${results[i]}`);
  }
})
